README.md

```diff
@@ -7,20 +7,22 @@ MOLGENIS is a collaborative open source project on a mission to generate great s
 See http://molgenis.github.io for documentation.
 
 ## Installation
-See [Getting Started Guide](http://molgenis.github.io/develop/start).
+See [Getting Started Guide](docs/develop/start.md).
```

Comment:
Not sure how, but ideally the github README.md links to the gitbook site but the gitbook site itself need not link to itself.